language/German/strings.po

```diff
@@ -2621,7 +2671,7 @@ msgstr "Schieberegler-Farbe"
 #: src/frontend/mame/ui/custui.cpp:620
 msgctxt "color-option"
 msgid "Graphics viewer background"
-msgstr ""
+msgstr "Hintergrund Grafikanzeige"
```

Comment:
"Hintergrund für..."?

Reply:
For this one I think @darkstar is right.

Reply:
Fixed